plutus-core/plutus-core/test/Value/Spec.hs

```diff
@@ -124,7 +185,12 @@ prop_deleteCoinPreservesInvariants v =
   vs = scanr (\(c, t, _) -> V.deleteCoin (V.unK c) (V.unK t)) v fl
 
 toPositiveValue :: Value -> Value
-toPositiveValue = V.pack . fmap (fmap abs) . V.unpack
+toPositiveValue =
+  V.pack . fmap (Map.map (\q ->
+    case V.quantity (abs (V.unQuantity q)) of
+      Just absQ -> absQ
+      Nothing   -> error $ "toPositiveValue: abs quantity out of bounds: " <> show (V.unQuantity q)
```

Comment:
Doesn't this mean that tests using `toPositiveValue` may randomly fail? It's better to use `maxBound` in this case.